### DISCUSSION TITLE
更新日志 - 20230512

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#768] [Announcements]
# 应用：
* QuickStart：
  1. 修复某些情况下首页显示的本地时间错误的问题 #737

# 设备：
* x86：支持了Ventoy多镜像U盘启动，7z/gzip解压后放在U盘里即可使用，无需额外处理镜像文件 #431
* rk3568: 
  1. 打开openssl优化选项，提高加解密性能
  2. 优化网卡CPU调度，提高性能和稳定性
  3. 增加22.03.4测试固件，支持部分无线网卡 （非此固件，需手动更新，[详情](https://github.com/istoreos/istoreos/issues/716)）

**Comment by jay5jiang** [comment]
什么时候可以让r5s突破千兆？

**Reply by jjm2473** [reply]
现在不行吗？

**Reply by jay5jiang** [reply]
> 现在不行吗？

不行，依然是500兆。。。

**Comment by Yangmeism** [comment]
为什么我不能拨号上网，光猫设置成桥接了，网线也插到ETH0上，就是拨不了号，用电脑试了能正常拨号

**Reply by jjm2473** [reply]
试试修改wan口的mac地址

**Reply by Yangmeism** [reply]
改了MAC地址后确实可以拨号上网，但还有个问题就是passwall和passwall2都无法使用，不知道是我机场的配置文件问题还是怎么，反正这两个插件只要开启后别说翻墙，就连国内网都连不上了，用openclash就可以正常使用

**Comment by hxpcyj** [comment]
能不能请老大搞个LNMP之类的插件

**Reply by jjm2473** [reply]
这些东西不是用docker轻轻松松搞定吗

**Reply by hxpcyj** [reply]
搞不定，试了安装了宝塔，数据库不能正常

**Reply by wcbspp** [reply]
docker确实轻松搞定 需要配置
